packages/share/example/ios/RunnerUITests/FLTShareExampleUITests.m

```diff
@@ -0,0 +1,43 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <XCTest/XCTest.h>
+
+@interface FLTShareExampleUITests : XCTestCase
+
+@end
+
+@implementation FLTShareExampleUITests
+
+- (void)setUp {
+  self.continueAfterFailure = NO;
+}
+
+- (void)testShareWithEmptyOrigin {
+  XCUIApplication* app = [[XCUIApplication alloc] init];
+  [app launch];
+
+  XCUIElement* shareWithEmptyOriginButton = [app.buttons
+      elementMatchingPredicate:[NSPredicate
+                                   predicateWithFormat:@"label == %@", @"Share With Empty Origin"]];
+  if (![shareWithEmptyOriginButton waitForExistenceWithTimeout:30]) {
```

Comment:
30 seconds seems really long.

Reply:
We used the same waiting length on scenario app tests https://github.com/flutter/engine/blob/master/testing/scenario_app/ios/Scenarios/ScenariosUITests/GoldenPlatformViewTests.m#L11
I think the rational is that on ci, the animation might take some time to run and might cause false negative to happen.
This returns immediately after finding the item, so I think setting time-out to be longer should be fine?